apps/voting/contracts/Voting.sol

```diff
@@ -174,6 +219,17 @@ contract Voting is IForwarder, AragonApp {
         _executeVote(_voteId);
     }
 
+    /**
+    * @notice Configure representative `representative` for the sender
+    * @param _representative Address of the representative to configure for the sender
+    * @param _allowed Boolean to tell whether the representative is being allowed or not
+    */
+    function setRepresentative(address _representative, bool _allowed) external isInitialized {
+        // TODO: can we validate sth here to avoid malicious people from spamming events?
```

Comment:
Yea, checking balances could drive to some undesired scenarios, for example, you wouldn't able to allow a representative if you transfer your tokens right after a vote was created, weird scenario tho. Since it doesn't seem harmful, I'd leave it as is.